addons/account_check_printing/models/account_payment.py

```diff
@@ -147,6 +153,8 @@ def action_post(self):
         for payment in self.filtered(lambda p: p.payment_method_id == payment_method_check and p.check_manual_sequencing):
             sequence = payment.journal_id.check_sequence_id
             payment.check_number = sequence.next_by_id()
+            if payment.move_id and payment.journal_id.check_manual_sequencing:
```

Comment:
`payment.journal_id.check_manual_sequencing` seems to be already implied by `filtered`